src/styles/card.scss

```diff
@@ -0,0 +1,113 @@
+@import './variables';
+@import './stars';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: $font-family;
+  background-color: $white;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: $white;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-family: Roboto;
```

Comment:
The font-family should be taken from the variables file for consistency. If 'Roboto' is the base font for the project, it should be defined in the variables file and reused here to avoid repetition and facilitate changes in the future.